paperparcel-compiler/src/main/java/paperparcel/PaperParcelAutoValueExtension.java

```diff
@@ -60,7 +60,7 @@
 
   @Override
   public IncrementalExtensionType incrementalType(final ProcessingEnvironment processingEnvironment) {
-    return IncrementalExtensionType.ISOLATING;
+    return IncrementalExtensionType.AGGREGATING;
   }
```

Comment:
This processor is still isolating I think. Just the other one that isn't

Reply:
Hmm, Isn't they use one another? I mean when you generating implementation for ```@AutoValue``` classes you rely on global ```TypeAdapters``` same way, right?

Reply:
Oh, I check source code and see that you just add ```@PaperParcelAnnotation``` and then main processor generate real implememntation, my bad.